lib/src/framework/models/testable_gestures.dart

```diff
@@ -21,7 +21,9 @@ class TestableGestures {
     this.widgetDoubleTap = TestableGestureAction.toggle_overlay,
     this.widgetForcePressEnd,
     this.widgetForcePressStart,
-    this.widgetLongPress = TestableGestureAction.open_test_actions_dialog,
+    this.widgetLongPress,
+    this.widgetLongPressMoveUpdate =
```

Comment:
I don't mind adding the new gesture, but don't want to change the default.  This new type of gesture is a lot harder for users to find / execute so I'd prefer applications decide to use it themselves rather than making it a framework default.

Reply:
I see why you probably made this change.  See the comment above about making a new test and a new page for the test.  In that test, you can pass a different set of gestures (including moving the `widgetLongPressMoveUpdate` to `widgetLongPress`) just for the `Testable` widgets on that page.

That will allow you to test the new gesture _and_ keep the core defaults / example as-is.